src/main/java/com/commercetools/pspadapter/paymentHandler/impl/PaymentHandleResponse.java

```diff
@@ -36,49 +36,57 @@
 
     private final String approvalUrl;
 
+    private final String responseBody;
```

Comment:
OK I will change, but `responseBody` means that it's a body of this `PaymentHandleResponse` class 😄 .

Reply:
`PaymentHandleResponse` **is** the _response body_. We don't need one more 'responseBody` inside our _response body_. This will be confusing.

More or less it has a sense to call it `paypalPlusResponseBody` if you want to put there value from Paypal Plus API response, but it also doesn't have much sense since could be changed in future.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/74/commits/e09dd2a84a60e6dd5b2c6ba243cce8d32efbd5c1